fourteen-testing-api.js

```diff
@@ -36,6 +36,21 @@ module.exports = function(file, api) {
     }
   }
 
+  // returns boolean for other unit tests with needs object argument
+  function findOtherUnitTestSetup(nodePath) {
+    let hasSetup = j(nodePath).find(j.ExpressionStatement, { expression: { callee: {  name: 'setupTest' } } } )
+    let isRouteTest = hasSetup
+      && j.ExpressionStatement.check(nodePath)
```

Comment:
Not something to hold the PR up for, but it might be nice having a little comment to the right of each line here to let future travelers know more easily what is going on here since this seems quite involved.

Reply:
Completely agreed! Hearty advice for most codemod authoring :).